packages/components/ng-ovh-order-tracking/src/translations/Messages_fr_FR.json

```diff
@@ -28,7 +28,7 @@
   "order_tracking_history_details_FRAUD_CHECK": "",
   "order_tracking_history_details_FRAUD_DOCS_REQUESTED": "",
   "order_tracking_history_details_FRAUD_MANUAL_REVIEW": "Un délai de traitement supplémentaire est nécessaire. Il dure en moyenne 48 heures. Le service client pourra éventuellement vous contacter.",
-  "order_tracking_history_details_FRAUD_REFUSED": "Les données liées à cette commande ne présentent pas les garanties nécessaires à la transaction. Nous vous invitons à la renouveler en utilisant un autre moyen de paiement.",
+  "order_tracking_history_details_FRAUD_REFUSED": "<p>Suite à une analyse des données liées à votre commande ou relatives à votre compte, il nous apparait que celles-ci ne présentent pas toutes les garanties nécessaires en termes de sécurité de la transaction. Afin de pouvoir procéder au traitement de votre commande, au paiement de celle-ci et à l'exécution du contrat, OVHcloud vous invite à renouveler votre achat en utilisant un autre <a href=\"{{ paymentUrl }}\">moyen de paiement</a>.</p><p>Vous pouvez également contacter un de nos conseillers via la <a href=\"{{ helpUrl }}\" target=\"_blank\" rel=\"external\">rubrique aide</a> de votre compte pour nous fournir les éléments utiles à l'aboutissement de la transaction. Dans le cas contraire, celle-ci ne sera pas prise en compte.</p>",
```

Comment:
is there a way you can get rid of <a> element in the content? may be by splitting the text to smaller chunks.

Reply:
@antleblanc @radireddy are you both ok that i keep the `<p>` tag inside so i can procede a translation request ?
my reasoning is that `<p>` is standard enough (same as `<a>`) to be used and has a semantic meaning (paragraph).
It also has the benefits of providing the full context of both sentences.

On the template side it's cleaner since i don't have to handle this special case by adding another div with ng-if

Reply:
I agree that for template logic point of view, it's way more easier to use this technique but as a Translator point of view, I found it really confusing.

Especially when it comes to deal with this part of the translation `</p><p>`.

We have no guarantee that the DOM couldn't be break here after the translation process.

Even if we are using `<a href="..">` tags into our existing translations, I would recommend as much as we can, to avoid using HTML tags in translations.